docs/getting-started/browser-not-supported/snippets/overlay.hbs

```diff
@@ -0,0 +1,49 @@
+---
+title: Browser not supported (Overlay)
+layout: demo.hbs
+---
+<div class="browser-not-supported-overlay">
+  <div class="browser-not-supported">
+    <header class="header">
+      <div class="header-main">
+        <div class="container">
+          <a href="#" class="header-brand">
+            <img class="header-brand-image" src="{{relative '/images/axa.svg'}}" />
+          </a>
+        </div>
+      </div>
+    </header>
+    <div class="container">
+      <div class="jumbotron">
```

Comment:
Wondering if you could put the `container` into the `jumbotron`, so the blue background is full-width, like our other designs?